metrics/benchmark_test.go

```diff
@@ -10,19 +10,39 @@ import (
 	"testing"
 )
 
+func BenchmarkNewTag(b *testing.B) {
+	b.Run("1 tag", func(b *testing.B) {
+		doTagBench(b, 1)
+	})
+	b.Run("10 tag", func(b *testing.B) {
+		doTagBench(b, 10)
+	})
+	b.Run("100 tag", func(b *testing.B) {
+		doTagBench(b, 100)
+	})
+}
+
+func doTagBench(b *testing.B, n int) {
```

Comment:
this `n` argument is getting ignored, is there supposed to be an inner loop?

Reply:
actually its probably more interesting to test different string lengths to see how that affects things (is a 10-char string 10x slower than a 1-char string?